Rakefile

```diff
@@ -55,23 +65,39 @@ task 'spec:setup' do
   ENV['RACK_ENV'] = ENV['LOKKA_ENV'] = 'test'
 end
 
-begin
-  require 'rspec/core/rake_task'
+unless Lokka.production?
```

Comment:
Without this, `LoadError` arise on production when `bundle exec rake console` are executed.